setup.py

```diff
@@ -117,12 +118,15 @@ def build_extensions(self):
             print("CC: " + os.environ.get('CC'))
             print("CXX: " + os.environ.get('CXX'))
         if sys.platform == 'darwin':
-            if os.system('which /usr/local/opt/llvm/bin/clang++ > /dev/null') == 0:
-                os.environ['CC'] = '/usr/local/opt/llvm/bin/clang'
-                os.environ['CXX'] = '/usr/local/opt/llvm/bin/clang++'
-            elif os.system('which /usr/local/opt/llvm@15/bin/clang++ > /dev/null') == 0:
-                os.environ['CC'] = '/usr/local/opt/llvm@15/bin/clang'
-                os.environ['CXX'] = '/usr/local/opt/llvm@15/bin/clang++'
+            brew_prefix = '/usr/local/opt'
+            if platform.machine() == 'arm64':
+                brew_prefix = '/opt/homebrew/opt'
```

Comment:
I am using a MacBook M1, but brew_prefix is `/usr/local/opt/`

Reply:
To obtain the homebrew prefix, one could use the command 'brew --prefix'.